packages/app/src/runner/event-manager.ts

```diff
@@ -163,6 +165,10 @@ export class EventManager {
       })
     })
 
+    this.ws.on('cross:origin:delaying:html', (request) => {
```

Comment:
original [event-manager](https://github.com/cypress-io/cypress/pull/18075/files#diff-fc3c1a5a6ff3660d9f29ce4238013c5b73e39497f86d9bc07e5bfcd5776cc91d) code in `runner-shared` package